akka-docs/src/main/paradox/common/cluster-usage.md

```diff
@@ -0,0 +1,166 @@
+
+Creating a cluster is about deploying two or more nodes and make then behave as if they were one single application. Therefore it's extremely important that all nodes in a cluster are configured with compatible settings. 
+
+The Configuration Compatibility Check feature ensures that all nodes in a cluster have a compatible configuration. Whenever a new node is joining an existing cluster, a subset of its configuration settings (only those that are required to be checked) is sent to the nodes in the cluster for verification. Once the configuration is checked on the cluster side, the cluster sends back its own set of required configuration settings. The joining node will then verify if it's compliant with the cluster configuration. The joining node will only proceed if all checks pass, on both sides.   
+
+New custom checkers can be added by extending `akka.cluster.JoinConfigCompatChecker` and including them in the configuration. Each checker must be associated with a unique key:
+
+```
+akka.cluster.configuration-compatibility-check.checkers {
+  my-custom-config = "com.company.MyCustomJoinConfigCompatChecker"
+}
+``` 
+
+@@@ note
+
+Configuration Compatibility Check is enabled by default, but can be disabled by setting `akka.cluster.configuration-compatibility-check.enforce-on-join = off`. This is specially useful when performing rolling updates. Obviously this should only be done if a complete cluster shutdown isn't an option. A cluster with nodes with different configuration settings may lead to data loss or data corruption. 
+
+This setting should only be disabled on the joining nodes. The checks are always performed on both sides, and warnings are logged. In case of incompatibilities, it is the responsibility of the joining node to decide if the process should be interrupted or not.  
+
+If you are performing a rolling update on cluster using Akka 2.5.9 or prior (thus, not supporting this feature), the checks will not be performed because the running cluster has no means to verify the configuration sent by the joining node, nor to send back its own configuration.  
+
+@@@ 
+
+
+## Cluster Management
+
+<a id="cluster-http"></a>
+### HTTP
+
+Information and management of the cluster is available with a HTTP API.
+See documentation of [Akka Management](http://developer.lightbend.com/docs/akka-management/current/).
+
+<a id="cluster-jmx"></a>
+### JMX
+
+Information and management of the cluster is available as JMX MBeans with the root name `akka.Cluster`.
+The JMX information can be displayed with an ordinary JMX console such as JConsole or JVisualVM.
+
+From JMX you can:
+
+ * see what members that are part of the cluster
+ * see status of this node
+ * see roles of each member
+ * join this node to another node in cluster
+ * mark any node in the cluster as down
+ * tell any node in the cluster to leave
+
+Member nodes are identified by their address, in format *akka.**protocol**://**actor-system-name**@**hostname**:**port***.
```

Comment:
problem from original text: remove the `**` from this (wrong formatting), and the protocol part should be removed
```
`akka://actor-system-name@hostname:port`
```